### DISCUSSION TITLE
VCRUNTIME140.dll was missing

### DISCUSSION THREAD

**Discussion by sahiljadhav123** [marcelotduarte/cx_Freeze#2417] [General]
Showing error when trying to run build file on another system Please help Sir
Thanks..

**Comment by marcelotduarte** [comment]
https://cx-freeze.readthedocs.io/en/stable/faq.html#microsoft-visual-c-redistributable-package

**Reply by sahiljadhav123** [reply]
Thanks Sir for reply.
But i have the package in my system and when i use this in build.exe then it work for 1 system and not for other showing this error then can u help me to set that path of dll file i can pass it seprate file no issues but any path option can u suggest to set that file

**Reply by marcelotduarte** [reply]
Using include_msvcr should work for all Windows 10 and 11. Some system are Windows 7 or another version?

**Reply by sahiljadhav123** [reply]
System is windows 10

**Reply by sahiljadhav123** [reply]
after using  include_msvcr error is showing = api-ms-in-core-path-l1-1-0.dll is missing from the computer 
please help Sir....

**Reply by sahiljadhav123** [reply]
windows 7 issue is  api-ms-in-core-path-l1-1-0.dll is missing from the computer
please help Sir....

**Reply by marcelotduarte** [reply]
If you are using older Windows version than Windows 10 and latest system updates are not installed, [Universal C Runtime](https://support.microsoft.com/en-us/help/2999226/update-for-universal-c-runtime-in-windows) might be also required.

**Reply by sahiljadhav123** [reply]
Okay Thanks Sir for Help.....
